src/main.rs

```diff
@@ -3,15 +3,43 @@ extern crate log;
 
 extern crate http;
 extern crate simple_server;
+use std::env;
 
 use http::header;
 use simple_server::{Method, Server, StatusCode};
 use std::fs;
+use std::path::PathBuf;
 fn main() {
     let host = "127.0.0.1";
-    let port = "5555";
+    let port = "5556";
 
-    let server = Server::new(|request, mut response| {
+    let mut path = PathBuf::new();
+
+    match env::current_exe() {
```

Comment:
I'm a little confused what's happening here.  It looks like a binary is being passed from the environment?

Reply:
The path of executable is passed. So i can determine relative to it the paths of my html and text files.
This was is just to ensure that running the server with `cargo run` or with ./executable behaves in the same way.

This code will be replaced by a more appropriate server soon as discussed, i.e. a static web server without the backend code.